src/QuadGK.jl

```diff
@@ -78,7 +78,11 @@ function evalrule(f, a,b, x,w,gw, nrm)
     Ig *= s
     E = nrm(Ik - Ig)
     if isnan(E) || isinf(E)
-        throw(DomainError())
+        @static if VERSION < v"0.7.0"
```

Comment:
The `if` statement is not needed, since QuadGK master required 0.7, and even if it didn't the 1-arg form of `DomainError` is supported by Compat.jl